tests/test_oml/test_ddp/test_accumulator.py

```diff
@@ -12,31 +12,61 @@
 @pytest.mark.long
 @pytest.mark.parametrize("world_size", [1, 2, 3])
 @pytest.mark.parametrize("device", ["cpu", "cuda"] if torch.cuda.is_available() else ["cpu"])
-def test_ddp_accumulator(world_size: int, device: str) -> None:
-    run_in_ddp(world_size=world_size, fn=check_ddp_accumulator, args=(device,))
+@pytest.mark.parametrize("create_duplicate", [True, False])
+def test_ddp_accumulator(world_size: int, device: str, create_duplicate: bool) -> None:
+    run_in_ddp(world_size=world_size, fn=check_ddp_accumulator, args=(device, create_duplicate))
 
 
-def check_ddp_accumulator(rank: int, world_size: int, device: str) -> None:
+@pytest.mark.parametrize("device", ["cpu", "cuda"] if torch.cuda.is_available() else ["cpu"])
+@pytest.mark.parametrize("create_duplicate", [True, False])
+def test_fake_ddp_accumulator(device: str, create_duplicate: bool) -> None:
+    # we expect the same behaviour without initializing DDP
```

Comment:
What is the goal of this setup?

Maybe you expect `the same behaviour without initializing DDP`, but I think [these guys](https://github.com/OML-Team/open-metric-learning/blob/ec091f183fc1f7182a4c28e9073bc1a01b7fa8cc/oml/ddp/utils.py#L7) from `torch` that `Accumulator` use expects a bit different setup when working with DDP 😁

Reply:
The goal of this setup is to check that duplicates can be removed in non DDP setup as well. For example, internal implementation of Accumulator may immediately remove `self` in `sync` method if `not is_ddp()` as it was before. However as far as we want to remove duplicated in any regime, we need to have a test for it.

Reply:
anyway, changed comment a bit:
` # we expect the same duplicate removing behaviour without initializing DDP`

Reply:
Got it. 
But I still have concern about *partially* initialized DDP. `check_ddp_accumulator` call `init_ddp` on first line where DDP process group is created. So in this case maybe `is_ddp` is `True`, but test works because `world_size` is 1. Could you add print and check it?

Reply:
i dont understand you
we don't run init_ddp in this test, we only run it for the real ddp test

Reply:
check where exactly ddp_init is